packages/components/src/RecurringSelect/RecurringSelect.tsx

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable import/no-internal-modules */
```

Comment:
had to do this here, and a couple similar ones

is this an acceptable solution, or is there a preferable way to solve this?

it is upset that we have this strcuture

```
RecurringSelect
--- WeeklySelect
--- MonthlySelect
--- OtherSelectComponents
```

then in this component we are importing them

Reply:
@darryltec any thoughts on how to best approach this? is the lint disable fine?